schema_utils.py

```diff
@@ -221,6 +240,17 @@ class _Validators(object):
     """
     @classmethod
     def get(cls, validator_id):
+        """Gets the value of the validator_id of the object.
+
+        Args:
+            validator_id: str. That contains the attribute's name.
+
+        Returns:
+            *. Returns the value of the name attribute of an object.
```

Comment:
Both the typeinfo and the description need to be clearer. I don't understand what this function is doing based on the description.

Reply:
Done